auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -37,11 +48,95 @@
                 <div>{{ displayStatus(item.status) }}</div>
               </template>
 
-              <!-- Displaying Formatted Date -->
+              <!-- Displaying Formatted Submission Date -->
               <template #[`item.submissionDate`]="{ item }">
                 <div>{{ formatDate(item.submissionDate) }}</div>
               </template>
 
+              <!-- Displaying Formatted Future Effective Date and Tooltips-->
+              <template #[`item.futureEffectiveDate`]="{ item }">
+                <div>
+                  <v-tooltip bottom>
+                    <template #activator="{ on, attrs }">
+                      <span
+                        v-bind="attrs"
+                        style="border-bottom: 1px dotted"
+                        v-on="on"
+                      >
+                        {{ formatDate(item.futureEffectiveDate) }}
+                      </span>
+                    </template>
+                    <span>{{ formatDate(item.futureEffectiveDate) }} at {{ formatTime(item.futureEffectiveDate) }} Pacific time</span>
+                  </v-tooltip>
+                  <IconTooltip
+                    v-if="expiredDate(item.futureEffectiveDate)"
+                    icon="mdi-alert"
+                    maxWidth="300px"
+                    colour="#D3272C"
+                    :iconStyling="{'font-size': '1.5em', 'margin-left': '4px'}"
+                    :location="{top: true}"
+                  >
+                    <div>
+                      <strong>Alert:</strong><br>
+                      <span> The Future Effective Date for this filing has passed.</span>
+                    </div>
+                  </IconTooltip>
+                  <IconTooltip
+                    v-if="item.futureEffectiveDate && daysLeft(item.futureEffectiveDate) <= 3"
```

Comment:
Should we always show the icon and tooltip for a FE date?

Reply:
Per Figma designs, we only show the icon and tooltip when the FE date passed, and when it's in 3 days, I think it makes sense?

Reply:
OK. Leave it. Resolving.